packages/material-ui-benchmark/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@material-ui/benchmark",
-  "version": "4.0.0",
+  "version": "4.1.0",
```

It's private, we don't release it

```suggestion
  "version": "4.0.0",
```